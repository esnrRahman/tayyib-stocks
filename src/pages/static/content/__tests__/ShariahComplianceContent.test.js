import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import ShariahComplianceContent from '../ShariahComplianceContent';

describe('<ShariahComplianceContent />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ShariahComplianceContent />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
