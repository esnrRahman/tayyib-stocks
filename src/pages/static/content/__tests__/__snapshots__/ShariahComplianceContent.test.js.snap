// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShariahComplianceContent /> renders correctly 1`] = `
<Fragment>
  <h1>
    Shari'ah Compliant Investments
  </h1>
  <p>
    Three principles need to be abided by when examining an investment for Shar’iah permissibility:
  </p>
  <ol>
    <li>
      The investment must be riba free
    </li>
    <li>
      The potential for "unethical concerns" such as gambling, or tobacco or alcohol in the investment
    </li>
    <li>
      The nature of the contract between the parties involved
    </li>
  </ol>
  <p>
    A two stage screening process may be applied to an investment to qualify it.
  </p>
  <h2>
    1. Industry Screen
  </h2>
  <p>
    The following areas should be avoided:
  </p>
  <ul>
    <li>
      Alcohol
    </li>
    <li>
      Non-Shar’iah Compliant Banks or other Financial Institutions
    </li>
    <li>
      Gambling
    </li>
    <li>
      Pornography
    </li>
    <li>
      Entertainment
    </li>
    <li>
      Tobacco
    </li>
    <li>
      Any business deriving greater than 5% of its income from the above sectors
    </li>
    <li>
      Any business whose activities are prejudicial to the interests of Islam or Muslims
    </li>
  </ul>
  <p>
    The Tayyib Stocks web application, unfortunately, cannot automate this step of the compliance process because automating this involves quite a complicated mechanism; even at the end of which the results cannot be 100% guaranteed to be reliable.
    <br />
    <br />
    However, to manually apply an industry screen, one of two methods maybe employed:
  </p>
  <ul>
    <li>
      <b>
        Simple approach: 
      </b>
      Apply a sector-based screen (i.e. any company falling into an impermissible sector, e.g. Entertainment, is disregarded)
    </li>
    <li>
      <b>
        Sophisticated approach: 
      </b>
      Even though a company may fall within a sector which is classified as non-compliant, the user may decide to -
      <br />
      Individually screen the stock using various methods that include consulting company statements, third-party sites and contacting the companies themselves.
    </li>
  </ul>
  <h2>
    2. Financial Ratio Screen
  </h2>
  <p>
    In order for an investment in a company to be halal, it has to have the following financial ratios less than
     
    <b>
      33%
    </b>
    :
  </p>
  <ul>
    <li>
      Total debt divided by most recent 24-month average market capital
    </li>
    <li>
      The sum of a company’s cash and interest-bearing securities divided by most recent 24-month average market capital
    </li>
    <li>
      Accounts receivable divided by most recent 24-month average market capital
    </li>
  </ul>
  <p>
    The Tayyib Stocks Web Application helps users to automatically do this step of the screening process.
    <br />
    Users can go to the Home Page of this application and just search the stock ticker to obtain a simple Pass/Fail result.
    <br />
    The actual values of the financial ratios mentioned here are also provided to the user for reference.
  </p>
  <p>
    These financial ratios help ensure that the majority of the dealings of the company is not riba-based. When a company takes debt, it has to usually pay interest back. Hence, the debt-to-capital ratio is considered. Similar arguments can be made for the other two ratios.
  </p>
</Fragment>
`;
